cg-producer/entrypoint.py

```diff
@@ -522,9 +65,13 @@ def consume(self):
                 release
             ))
 
-            generator = CallGraphGenerator(self.out_topic, self.err_topic,
-                    self.source_dir, self.producer, release)
-            generator.generate()
+            generator = CallGraphGenerator(self.source_dir, release)
+            output = generator.generate()
+
+            if output["Status"]=="Success":
```

Comment:
You can use constants instead of strings. Makes the code more maintainable, e.g.:

```
SUCCESS = "Success"
...
if output["Status"] == SUCCESS:
...
```

Reply:
You are right, changed on [8bbdc5f](https://github.com/fasten-project/pypi-tools/pull/6/commits/8bbdc5f69710d4942fb0a4257dee02b97caaa797).